lib/internal/test_runner/test.js

```diff
@@ -760,8 +760,10 @@ class Suite extends Test {
 
     try {
       const { ctx, args } = this.getRunArgs();
+      const runArgs = ArrayPrototypeSlice(args);
+      ArrayPrototypeShift(runArgs, ctx);
```

Comment:
Presumably skipping the slice operation would be faster.
```suggestion
      const runArgs = [this.fn, ctx];
      ArrayPrototypePushApply(runArgs, args);
```